packages/tools/client-debugger/client-debugger-chrome-extension/src/react-components/TelemetryView.tsx

```diff
@@ -50,10 +60,17 @@ export function TelemetryView(): React.ReactElement {
 
 		messageRelay.on("message", messageHandler);
 
+		// Request all log history
+		messageRelay.postMessage({
+			source: extensionMessageSource,
+			type: "GET_TELEMETRY_HISTORY",
+			data: { contents: [] },
```

Comment:
```suggestion
			data: undefined,
```
Can we do that? Since this particular message has no data/contents. Same for the one in client-debugger-view.